lib/runtime2/threepio/trace/TraceManager.ts

```diff
@@ -0,0 +1,148 @@
+import util from 'util';
+import { TimePoint, TimePointOptions } from './TimePoint';
+import { TaskFlowSpan, SpanContext } from './TaskFlowSpan';
+
+interface CallNode {
+  requestId: string;
+  TaskFlowSpan: TaskFlowSpan[];
```

Comment:
```suggestion
  span: TaskFlowSpan[];
```
另外，CallNode 应该直接依赖 Span 吧？还有这里为什么是一个数组？